config/chime_rfi.yaml

```diff
@@ -263,55 +280,59 @@ gpu:
     updatable_rfi_zeroing: /rfi_masking/toggle
```

Comment:
Why is this called toggle if it gets a boolean?

Reply:
oh it looks like `chime_rfi.yaml` is out of data... for now just look at `chime_science_run_gpu.yaml`.   I think you can have a "toggle" as a boolean.  According to google define:  `2. COMPUTING a key or command that toggles between two modes, such as the caps lock key.`